src/audio/base_fw.c

```diff
@@ -71,7 +81,11 @@ static int basefw_config(uint32_t *data_offset, char *data)
 
 	/* TODO: add log support */
 	tuple = next_tuple(tuple);
-	set_tuple_uint32(tuple, IPC4_TRACE_LOG_BYTES_FW_CFG, 0);
+#ifdef CONFIG_LOG_BACKEND_ADSP_MTRACE
+	log_bytes_size = 1024;
```

Comment:
Need to put this magic number somewhere (or actually, get from some runtime interface).

Reply:
The thing with the LOG_BYTES definition: in some platforms it is set to the _whole_ size of the log window (1024 * number of slots), on some platforms it is set to the size of _one_ slot (1024).
The only consistency is that it is _set_ if mtrace type of log extraction is available, thus the kernel ignores the value itself and checks the slot mapping page to find the slots.

Reply:
iow, setting it to 1024 is the right thing if mtrace is usable.

Reply:
#define MAILBOX_TRACE_SIZE	SRAM_TRACE_SIZE ?

Reply:
Ack, I'll pick some define for this. In theory this value should come from Zephyr side and code on SOF side should be hw agnostic.